src/payments/dtos/generate-certificate-request.dto.ts

```
@@ -0,0 +1,39 @@
+  courseId: string;
+
+  @ApiProperty({
+    example: '2025-12-12T00:00:00.000Z',
+    description: 'Certificate issuance date (ISO 8601 string passed through to certificate service)',
+  })
+  @IsNotEmpty()
+  @IsString()
+  issuanceDate: string;
+
+  @ApiProperty({
+    example: '0000-00-00T00:00:00.000Z',
```

ESLint

[error] Replace `'0000-00-00T00:00:00.000Z'` with `"0000-00-00T00:00:00.000Z"`

(prettier/prettier)